app/src/main/java/com/nervousfish/nervousfish/service_locator/IServiceLocator.java

```diff
@@ -25,4 +25,11 @@
     INFCHandler getNFCHandler();
     IQRHandler getQRHandler();
 
+    /**
+     * Registers the class specified to the EventBus
+     *
+     * @param object The Object that should be registed to the EventBus
+     */
+    void registerToEventBus(final Object object);
```

Comment:
Within a few days it will be used, for example to let the BluetoothActivity know that there were new Bluetooth devices detected or to let the NFC Activity know that some device touched your OnePlus 3